INSTALL.md

```diff
@@ -259,6 +259,13 @@ Synapse via https://packages.matrix.org/debian/. They are available for Debian
 
 ```
 sudo apt install -y lsb-release wget apt-transport-https
+```
+for Debian/Ubuntu versions shipping `apt` < 1.6 (Debian < 10 / Ubuntu < 18.04), or
```

Comment:
Could you add empty lines between code blocks and normal text please? This helps with readability when doing changes to the document.

Reply:
Done.